### PR TITLE
Custom line circle radius

### LINKED ISSUE #150: Line chart - single continuous plot instead of points
#### I am submitting a...

- [X] Feature request
- [ ] Design defect
- [ ] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:
0.8.3
### Issue description
graph needs to have 30 plot points to indicate last 30 days (below does not indicate 30days, but hopefully illustrates the point). We need a single continuous plot instead of points.
a

This is what is desired:
b

### PR DESCRIPTION
**Pull request by theiliad**
Closes #150

### CODE REVIEW

File: packages/core/demo/demo-data/line.ts [hunk 1]
```diff
@@ -130,6 +130,9 @@ export const lineOptions = {
 			]
 		}
 	},
+	points: {
```
Comment (zvonimirfras) [inline packages/core/demo/demo-data/line.ts]: We shouldn't replace our demo with this. We can have a separate demo if needed.

File: packages/core/src/configuration.ts [hunk 2]
```diff
@@ -12,6 +12,10 @@ const baseOptions: any = {
 	tooltip: {
 		formatter: null
 	},
+	// Only used for line chart
```
Comment (zvonimirfras) [inline packages/core/src/configuration.ts]: They shouldn't be in baseOptions then.

File: packages/core/src/line-chart.ts [hunk 3]
```diff
@@ -35,6 +35,15 @@ export class LineChart extends BaseAxisChart {
 		}));
 	}
 
+	getCircleRadius() {
+		return this.options.points.radius || Configuration.charts.pointCircles.radius;
```
Comment (zvonimirfras) [inline packages/core/src/line-chart.ts]: Why different naming for the same thing in options and configuration? points vs pointCircles?

File: packages/core/demo/demo-data/line.ts (hunk 1 above)
Comment (theiliad) [reply packages/core/demo/demo-data/line.ts]: Why wouldn't we? We allow custom radius of the points now, how else would someone find out?

Comment (zvonimirfras) [reply packages/core/demo/demo-data/line.ts]: Not that we shouldn't demo the functionality, we shouldn't _replace_ defaults with a custom value without making it obvious it's a customization.
Having an interactive circle size demo or a separate demo for the functionality.
Documentation.

**Pull request merged**